src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -3,19 +3,18 @@
 import edu.wpi.first.wpilibj.Preferences;
 
 public class MoPrefs {
-  static final double INTAKE_ROLLER_SETPOINT = 1.0;
-  /**
-   * The maximum accleration of the intake roller, in motor setpoint per scheduler
-   * cycle.
-   */
+  static final double INTAKE_ROLLER_SETPOINT = -0.125;
```

Comment:
Wow, we're really slowing down the intake.  I'm thinking we chose the wrong gearboxes.

Also, this should be a positive value.  If the motors run backwards, they should be inverted in the subsystem.